rllib/tuned_examples/pendulum-sac.yaml

```diff
@@ -0,0 +1,37 @@
+# Pendulum SAC can attain -150+ reward in 5-6k timesteps (*disregarding initial exploration steps like in softlearning codebase)
+# Configurations are the similar to original softlearning/sac codebase

```

Comment:
Can you update the pendulum-sac regression test file? (assuming it is reliable enough for that?)

Reply:
Done!